test/index.html

```diff
@@ -12,7 +12,7 @@
     <link rel="stylesheet" href="assets/css/style.css">
   </head>
   <body>
-
+//just for a test
```

Comment:
<training> HTML comments don't work with `//` --- they need to be in the format `<!-- your comment -->`. Please fix.